src/main/java/microsoft/exchange/webservices/data/ExchangeService.java

```diff
@@ -3833,11 +3837,11 @@ public void setEnableScpLookup(boolean value) {
    * should use "Exchange2007" as the server version String rather than
    * Exchange2007_SP1. </remarks>
    */
-  protected boolean getExchange2007CompatibilityMode() {
+  public boolean getExchange2007CompatibilityMode() {
```

Comment:
please split javadoc for getter and setter since it will not be displayed in setExchange2007CompatibilityMode.